README.md

```diff
@@ -46,21 +46,36 @@ pip install appmap
 For projects that use `poetry` , add the `appmap` package to `pyproject.toml`.
 
 ## Configuration
-Add your modules as `path` entries in `appmap.yml`:
+Add your modules as `path` entries in `appmap.yml`, and external packages
+(distributions) as `dist`:
 
 ```yaml
 name: my_python_app
 packages:
 - path: app.mod1
+  shallow: true
 - path: app.mod2
   exclude:
   - MyClass
   - MyOtherClass.my_instance_method
   - MyOtherClass.my_class_method
+- dist: Django
+  exclude:
+  - django.db
 ```
 
 Note that `exclude`s are resolved relative to the associated path. So, for example, this
-configuration excludes `app.mod2.MyClass`
+configuration excludes `app.mod2.MyClass`.
+
+`dist` specifiers use importlib.metadata, and so should work regardless of
```

Comment:
Perhaps, but packages and distributions are not the same thing; distributions do not always come from packages and packages not always contain exactly one distribution. Python doesn't know about packages and there are many tools and formats of packaging, and probably more will be invented. I don't think it's a good idea to interact with specific packaging tools; thing that Python knows about is a "distribution" and calling it something else may lead to confusion ;)

Reply:
Isn't the value of `package` what I said: what  you'd pass to `pip install`, or put in `pyproject.toml` as a dependency? 

Maybe my confusion is understandable: https://pypi.org/ , https://packaging.python.org/glossary/#term-Distribution-Package ? ;) Also:
```sh
ajp@Alans-MacBook-Pro appmap-python % pip --help

Usage:
  pip <command> [options]

Commands:
  install                     Install packages.
  download                    Download packages.
  uninstall                   Uninstall packages.
  freeze                      Output installed packages in requirements format.
  list                        List installed packages.
```

So, while `dist` may be technically correct, it seems like most users are probably going to know it this thing as a `package`.

Reply:
Do we have a pythonista we could ask? I feel we're both dabbling ;)

Reply:
No, you and I are basically it, at least for quick turnaround.

I guess we can go with `dist`. If we do, though, I think this paragraph needs to be expanded to introduce the concept more gently:
> Also, I think we need to help with user understand what the value ~package~ `dist` can be: it's the string you'd use for `pip instal`l, or you'd add as a dependency in `pyproject.toml`. I know that's what "specifiers use importlib.metadata" implies, but it feels like it would be better to really spell it out.